### PR TITLE
[release-4.12] WINC-1177: Update version to 7.2.1

### PR DESCRIPTION
**Pull request by mansikulkarni96**
This commit was generated by hack/pre-release.sh
  Commands run: 
  hack/pre-release.sh release-4.12 z

### CODE REVIEW

**Review by jrvaldes — APPROVED** [review]
@mansikulkarni96  Thanks for working on this.  LGTM

Consider including the script arguments you used in the description of the PR

For example: hack/pre-release.sh release-4.12 z

File: bundle/manifests/windows-machine-config-operator.clusterserviceversion.yaml [hunk 1]
```diff
@@ -12,11 +12,11 @@ metadata:
     operatorframework.io/suggested-namespace: openshift-windows-machine-config-operator
     operators.openshift.io/valid-subscription: '["Red Hat OpenShift support for Windows
       Containers"]'
-    operators.operatorframework.io/builder: operator-sdk-v1.15.0+git
+    operators.operatorframework.io/builder: operator-sdk-v1.32.0
```
Comment (sebsoto) [inline bundle/manifests/windows-machine-config-operator.clusterserviceversion.yaml:15]: This shouldnt change. Release-4.12 still uses 1.15 osdk

**Review by sebsoto — APPROVED** [review]

**Pull request merged**